src/clients/ayast_setup.rb

```diff
@@ -95,6 +96,11 @@ def Setup
       Wizard.CreateDialog
       Mode.SetMode("autoinstallation")
       Stage.Set("continue")
+
+      # IPv6 settings will be written despite the have been
+      # changed or not. So we have to read them at first.
```

Comment:
I would prefer to have the read in:

https://github.com/yast/yast-network/blob/master/src/modules/Lan.rb#L81. 

But for the time being and the concrete bug, we could just add a <i><b>FIXME:</b> move to a better place and remove the Lan import dependency.</i>